edb/schema/constraints.py

```diff
@@ -766,13 +766,6 @@ def _populate_concrete_constraint_attrs(
                 attrs['subjectexpr'] = base_subjectexpr
                 inherited['subjectexpr'] = True
 
-        errmessage = attrs.get('errmessage')
-        if not errmessage:
-            errmessage = constr_base.get_errmessage(schema)
-            inherited['errmessage'] = True
-
-        attrs['errmessage'] = errmessage
-
```

Comment:
I'm a bit puzzled how `errmessage` gets inherited from abstract constraints now

Reply:
Through the normal generic inheritance mechanisms, right?

As far as I could tell, errmessage doesn't need any special handling